splitio/tests/test_clients.py

```diff
@@ -1155,80 +1155,80 @@ def test_test_killed_not_in_segment_key(self):
             self.fake_id_not_in_segment, 'test_killed'))
 
 
-class LocalhostEnvironmentClientParseSplitFileTests(TestCase, MockUtilsMixin):
```

Comment:
these tests fail because there's a .split file missing in localhost, for some reason it's not being mocked correctly and it looks for that file on the user's home. This is an old flakey test, not introduced by your code. We can work together to fix this

Reply:
mocked thread for these tests